static_config.yaml

```diff
@@ -1,9 +1,6 @@
 ---
 # This configuration should only have static settings.
 
-# 3PlayMedia FTP host for old workflow
-threeplay_ftphost: ftp.3playmedia.com
```

Comment:
@Qubad786 do we also need to remove this from veda-secure?

Reply:
Yes.

Reply:
we also need to remove this from `instance_config.yaml` in code.

Reply:
Done!